addons/realisticnames/stringtable.xml

```diff
@@ -3950,5 +3950,549 @@
             <Chinesesimp>口径: 5.7mm&lt;br /&gt;发数: 50&lt;br /&gt;使用于: P90</Chinesesimp>
             <Korean>구경: 5.7mm&lt;br /&gt;장탄수: 50&lt;br /&gt;사용됨: P90</Korean>
         </Key>
+        <Key ID="STR_ACE_RealisticNames_arifle_AKM">
+            <Chinese>AKM</Chinese>
+            <French>AKM</French>
+            <Spanish>AKM</Spanish>
+            <Italian>AKM</Italian>
+            <Polish>AKM</Polish>
+            <Russian>AKM</Russian>
+            <German>AKM</German>
+            <Czech>AKM</Czech>
+            <English>AKM</English>
```

Comment:
Move English to first position in stringtable. We forgo the second English entry Original, because it is a duplicate, however, this means the first language is the fall back if the game language is missing.
Therefore English should be at first position.